src/PrestaShopBundle/Resources/config/routing/admin/configure/shop_parameters/meta.yml

```diff
@@ -19,6 +19,22 @@ admin_meta_list_create:
         _controller: 'PrestaShopBundle:Admin\Configure\ShopParameters\Meta:create'
         _legacy_controller: AdminMeta
 
+admin_meta_list_save_form:
```

Comment:
naming is very different everywhere, you can check the conventions im trying to push https://github.com/PrestaShop/PrestaShop/issues/10597